vars/demoEnvPipeline.groovy

```diff
@@ -20,23 +18,23 @@ def call(playbookToRun) {
                 steps {
```

Comment:
Is it worth splitting in two stages at all, if the first stage only sets some variables?

Reply:
Not really, I will merge these stages.